modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -85,5 +85,34 @@ def report msg, response, error_only
       logger.error "Dnscmd failed:\n" + (response.is_a?(Array) ? response.join("\n") : "Response was not an array! #{response}")
       raise Proxy::Dns::Error.new("Unknown error while processing '#{msg}'")
     end
+
+    def match_zone(fqdn)
+      dns_zones = ::Proxy::Dns::Plugin.settings.dns_zones || ['bar.domain.local', 'domain.local', '33.168.192.in-addr.arpa', '168.192.in-addr.arpa']
```

Comment:
You'll need to use ```setup``` method which is called before every test to define plugin settings in tests:

```
def setup
  Proxy::Dns::Plugin.load_test_settings(
        :dns_server => 'a_server',
        :dns_zones => ['bar.domain.local', 'domain.local', '33.168.192.in-addr.arpa', '168.192.in-addr.arpa'])
end
```